src/datalinker/templates/links.xml

```diff
@@ -49,24 +49,32 @@
  </RESOURCE>
  <RESOURCE ID="cutout-sync" type="meta" utype="adhoc:service">
    <GROUP name="inputParams">
-     <PARAM arraysize="*" datatype="char" name="ID" ucd="meta.id;meta.main" 
+     <PARAM arraysize="*" datatype="char" name="ID" ucd="meta.id;meta.main"
```

Comment:
The SODA standard specifies, in [Erratum 1 to v1.0](https://wiki.ivoa.net/twiki/bin/view/IVOA/SODA-1_0-Err-1), that the UCD should be "meta.id;meta.dataset"

Reply:
I'll get this in a separate PR.